19_leaflet1/main.js

```diff
@@ -0,0 +1,14 @@
+// create a new map object, pass the <div> id as parameter:
+let mymap = L.map('mapid');
+// center it on ReDI school, zoom level 13:
+mymap.setView([52.531446, 13.384752], 13);
+
+// create a new layer
```

Comment:
What about adding an extra slide explaining what a layer is in order to make the transition easier?